new-packaging/src/debian/skeleton/debian/neo4j.default

```diff
@@ -7,10 +7,11 @@
 #
 #     systemctl edit neo4j.service
 #
+# This is automatically done upon upgrade.
 ######################
 #
 # Number of seconds to wait for shutdown before killing the process.
 #NEO4J_SHUTDOWN_TIMEOUT=120
 #
 # Max number of open file descriptors
-NEO4J_ULIMIT_NOFILE=40000
+#NEO4J_ULIMIT_NOFILE=60000
```

Comment:
Value: 
I thought I was being consistent with what we shipped in the release 3.1.0 package but looking at the [source](https://github.com/neo4j/neo4j/commit/1fc3c0ab9f352f883a60cb126d25496a6f61d9e6) it seems I was mistaken.

Commented:
Being commented out by default is to prevent an unnecessary migration to happen when you go from 3.2.0 to 3.2.0+, it was also commented out before, so in the final diff this is not a change.